### DISCUSSION TITLE
ipv4/ipv6 address filters

### DISCUSSION THREAD

**Discussion by Izolant** [nette/latte#367] [General]
Hello,

Is there a plan to add support for IP address filtering?

Something like:
https://fiddle.nette.org/latte/#4e3b1086c4

https://github.com/mlocati/ip-lib
https://docs.ansible.com/ansible/3/user_guide/playbooks_filters_ipaddr.html

**Comment by deba12** [comment]
You can make your own filter with such functionality.
I have made such simple filter for binary ip addresses.

```php
    /**
     * @param string|null $ip
     * @return string
     * @noinspection PhpUnused
     */
    public static function inet_ntop(?string $ip): string {
        if(empty($ip)){
            return '';
        }

        return inet_ntop($ip);
    }
```

with ```s1lentium/iptools:dev-master``` you can fulfill all of the desired functionality.
